overnight-battery-loan/form.tex

```diff
@@ -0,0 +1,48 @@
+\documentclass[a4paper, 12pt, landscape]{article}
+\usepackage[utf8]{inputenc}
+
+\usepackage[default,osfigures]{opensans}
+
+\usepackage[margin=0.5in]{geometry}
+\usepackage{tabularx}
+\setlength{\parindent}{0cm}
+
+\newcolumntype{Y}{>{\centering\arraybackslash}X}
+
+\begin{document}
+
+\begin{center}
+
+    {\huge{Overnight Battery Loan Form}}
+
+\end{center}
+
+Teams often request a battery and charger for use overnight to test their robots. Provide them with at most 2 batteries, a battery bag and a charger, and note down the part numbers.
+
+\begin{center}
+\begin{table}[h]
+\def\arraystretch{1.5}
+\begin{tabularx}{\textwidth}{|Y|Y|Y|Y|Y|Y|Y|Y|Y|}
+    \textbf{TLA} & \textbf{Charger Part Number} & \textbf{Battery Bag Part Number} & \textbf{Battery 1 Part Number} & \textbf {Battery 2 Part Number} & \textbf{Charger Returned?} & \textbf{Battery Bag Returned?} & \textbf{Battery 1 Returned?} & \textbf{Battery 2 Returned?} \\
```

Comment:
This table is pretty wide, how about re-working the returned boxes to being inline?

Reply:
I've put a version of this in https://github.com/srobo/kit-coordination-docs/pull/7